src/main/java/com/google/neighborgood/servlets/TaskServlet.java

```diff
@@ -14,17 +14,8 @@
 
 package com.google.neighborgood.servlets;
 
-import com.google.appengine.api.datastore.Cursor;
-import com.google.appengine.api.datastore.DatastoreService;
-import com.google.appengine.api.datastore.DatastoreServiceFactory;
-import com.google.appengine.api.datastore.Entity;
-import com.google.appengine.api.datastore.EntityNotFoundException;
-import com.google.appengine.api.datastore.FetchOptions;
-import com.google.appengine.api.datastore.Key;
-import com.google.appengine.api.datastore.KeyFactory;
-import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.*;
```

Comment:
Actually there's a reason why I import all these classes one by one instead of using a wild card. https://stackoverflow.com/a/147461
So personally, I am against the change here.

Reply:
This was actually done automatically by the IDE I used at the moment 😆 I'll edit these to go back to what it was previously.